### DISCUSSION TITLE
Is it possible to update composer.lock only?

### DISCUSSION THREAD

**Discussion by rbel-io** [renovatebot/renovate#24134] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

Latest

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Is it possible to create PR's with only a composer.lock file update? 

### What you tried so far.

Searched the docs I found only the opposite with lockFileMaintenance  

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
It's possible
